src/Table/CustomTable.stories.js

```diff
@@ -67,7 +71,7 @@ function dynamicHeightRenderer({
         <div
           onLoad={measure}
           ref={registerChild}
-          style={{ "overflow-wrap": "break-word", whiteSpace: "normal" }}
+          style={{ "overflow-wrap": "break-word", "whiteSpace": "normal" }}
```

Comment:
```suggestion
          style={{ "overflow-wrap": "break-word", whiteSpace: "normal" }}
```
Small thing, I think inline styles either have to be spelled normally how it would be in css e.g. `"white-space"` but React can convert it if camelCase is used e.g. `whiteSpace` without the quotes. 

Other than running `npm run build` and committing the new build file, everything LGTM